src/main/scala/wdl4s/values/WdlBoolean.scala

```diff
@@ -8,6 +8,7 @@ object WdlBoolean {
   val True = new WdlBoolean(true)
   val False = new WdlBoolean(false)
   def apply(value: Boolean) = if (value) WdlBoolean.True else WdlBoolean.False
+  def unapply(b: WdlBoolean): Option[Boolean] = Option(b.value)
```

Comment:
I added that because I needed to use `WdlBoolean` in a `match` statement and withoutt his it wouldn't work.  http://danielwestheide.com/blog/2012/11/21/the-neophytes-guide-to-scala-part-1-extractors.html